catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -152,6 +152,145 @@
             <artifactId>common-system</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.luxdb</groupId>
+            <artifactId>lux</artifactId>
```

Comment:
This is the old version of lux that we don't use anymore. We are using codice's forked version now in place of that. 

It should be this:
	    <dependency>
			<groupId>org.codice</groupId>
			<artifactId>lux</artifactId>
			<version>${lux.version}</version>
		</dependency>